impress-org/givewp
# Fix: Include Donor created date column in Donor csv export

Resolves #6825

## Description
This PR addresses the missing Donor Created Date column from the Donor exported csv.

It introduces the missing column by including the column name in the list of columns ( `method: csv_col()` ) & by including it in the donor query  ( `method: get_data()` )

## Affects

Donor exports

## Visuals


https://github.com/impress-org/givewp/assets/75056371/7ce4ee73-bcf7-41ef-80a9-87ae64107f48

## Testing Instructions

1. Go to Donations -> Tools
2. Select Export Donor -> Generate csv
3. View the downloaded csv
4. View the csv, you will see that it now has a donor created column

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204816624221913
